packages/headless/doc-parser/src/entity-builder.ts

```diff
@@ -4,7 +4,7 @@ import {AnyEntity, Entity, FuncEntity, ObjEntity} from './entity';
 import {emitAsTsDoc} from './tsdoc-emitter';
 
 interface Comment {
-  comment: DocComment | undefined;
+  comment: DocComment | string | undefined;
```

Comment:
Could we avoid the `string` type here? By leveraging the `DocComment`, we get access to a parsed structure, which will be useful as the project evolves to e.g. read the values of custom decorators.

Edit: I added a comment below related to this.